playwright/tests/e2e/features/transaction-overview.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Check Transaction Cardano Explorer
+
+  Scenario: Transactions section content
+    Given the user is in the general dashboard page in explorer portal test
+    When the user selects the Transactions option inside the Blockchain drop down menu
+    Then the user should see the transactions page containing the Search bar and Transactions table
+
+  Scenario: Transaction info widget content
+    Given the user is in the transactions page in explorer portal site
+    When the user selects any data of the row execpt the transaction hash, block number, epoch number or addresses hash of one of the transactions record in the table
+    Then the user should see the Info widget data with the respective data of the selected transaction
+
+  Scenario: Transaction detail page through transaction hash
+    Given the user is in the Transactions page 
+    When  the user selects the transaction hash of one record of the transactions table in the transactions page
+    Then the transaction details page of the select transaction should be opened
+    
+  Scenario: Block detail page through block number
+    Given the user is in the Transactions page portal
+    When  the user selects the block number of one record of the transactions table in the transactions page
+    Then the user should be redirected to the block detail page of the selected block number
+
+  Scenario: Epoch detail page through epoch number
+    Given the user is in the Transactions page 
+    When  the user selects the epoch number of one record of the transactions table in the transactions page
+    Then the user should be redirected to the epoch detail page of the selected epoch number
+
+  Scenario: Address detail view page through input/output address  
+    Given the user is in the Transactions page for go to address detail page
+    When the user selects the input or output address of one record of the transactions table in the transactions page
+    Then the user should be redirected to the address details page of the selected address hash
+
+  Scenario: Transaction detail page through info widget 
+    Given the user is in the Transactions page for go to transaction detail from detail button in widget
+    Given the user open the info widget of one transaction record in the transactions table of transaction page
+    When the user selects the view details button in the transcation info widget
+    Then the transaction details page of the selected transaction should be opened
+
+  Scenario: Address detail view page through info widget 
+    Given the user is in the Transactions page to address detail from detail button in widget
+    Given the user open the info widget of one transaction record in transaction web
```

Comment:
Again avoid repeat `Given` expression and use `And` expression, besides that this step execute the same action that the step of the previous test where you basically open the widget of a transaction, please keep in mind this I know that maybe in the document I wrote it a bit different but at the end is the same action so we can reuse steps and avoid duplicated code.